config/config.hocon.sample

```diff
@@ -0,0 +1,77 @@
+{
+  # Optional, but recommended
```

Comment:
It's optional if `AWS_REGION` is set or it's running on EC2, but recommended because SDK, KCL and connectors have different functions for determining endpoints and it can fail for some, like `cn-` regions.